include/host1x.h

```diff
@@ -161,12 +161,12 @@ int host1x_overlay_set(struct host1x_overlay *overlay,
 
 #define HOST1X_BO_CREATE(host1x, size, flags)				\
 ({									\
-	struct host1x_bo *bo = host1x_bo_create(host1x, size, flags);	\
-	if (!bo)							\
+	struct host1x_bo *_bo_ = host1x_bo_create(host1x, size, flags);	\
+	if (!_bo_)							\
 		fprintf(stderr,						\
 			"ERROR: %s:%d: host1x_bo_create() failed\n",	\
 			__FILE__, __LINE__);				\
-	bo;								\
+	_bo_;								\
 })
```

Comment:
I think the state of current error-handling in grate is good enough, at least you'll now if something gone wrong and where to start to look at. Grate is mostly a dev tool, graceful error-handling and avoiding of a tiny memleaks is always good to have but not necessary here IMO.